Source/SteveSloth/Private/MyEmuEnemyClass.cpp

```diff
@@ -1,7 +1,7 @@
 /****************************************************************************************
  * Copyright: SteveSloth
  * Name: Elad Saretzky
- * Script: MyEmuEnemyClass.h
+ * Script: MyEmuEnemyClass.cpp
```

Comment:
I can change this in my branch cause I'm working on the emu script, just so there's no conflicts